sdk/storage/azblob/README.md

```diff
@@ -0,0 +1,362 @@
+# Azure Storage Blob SDK for Go
+
+## Introduction
+
+The Microsoft Azure Storage SDK for Go allows you to build applications that takes advantage of Azure's scalable cloud
+storage. This SDK replaces the previously previewed [azblob package](https://github.com/azure/azure-storage-blob-go) in the separate repository.
+
+## Getting Started
+
+The Azure Blob SDK can access an Azure Storage account.
+
+### Prerequisites
+
+* Go versions 1.16 or higher. If you don't already have it, install [the Go distribution](https://golang.org/dl/)
```

Comment:
```suggestion
* Go versions 1.16 or higher
```

Reply:
It is the same in the previous SDK so I won't mind leaving it as it is.